scripts/build-node-scripts.sh

```diff
@@ -4,8 +4,10 @@ BABEL=$PWD/node_modules/.bin/babel
 
 mkdir -p $OUTDIR/server/js &&
 mkdir -p $OUTDIR/shared/js &&
+mkdir -p $OUTDIR/shared/partials &&
```

Comment:
I like the idea of splitting templates into partials and full templates. Thanks!

Reply:
Thx!